gedi_test.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# Define project IDs, end years, and buffer sizes
+projects=("1650")
+end_years=("2021" "2022" "2023")
+buffer_sizes=("70" "80" "90" "100")
+
+# Base directories
+project_dir="/maps/jh2589/projects"
```

Comment:
If anyone other than you runs this scripts they're going to write to your directory. Can you make the script take an argument that is then used instead of `/maps/jh2589`?

Reply:
Yep this was a mistaken push of an experimental script